### PR TITLE
Django 3.0

### PR DESCRIPTION
**Pull request by erikvw**
Several minor fixes to make compatible with Django 3.0

### CODE REVIEW

**Comment by erikvw** [comment]
I removed Django from install requires and expect the user to provide their version of Django. Is this OK?

**Comment by damon-rand** [comment]
@erikvw Thanks for the PR. Sorry for the lag in replies... but we are a bit preoccupied. I will tag a few more people for review and see if we can get this merged.

File: .gitignore [hunk 1]
```diff
@@ -12,5 +12,6 @@ eggs
 parts
 MANIFEST
 multisite/*.egg-info
+.coverage
```
Comment (jordiecometrica) [inline .gitignore]: This change seems unrelated to this commit, but I can fix this in-flight.

File: multisite/models.py [hunk 2]
```diff
@@ -5,6 +5,8 @@
 import operator
```
Comment (jordiecometrica) [inline multisite/models.py:5]: There's a typo in the commit message, but I can fix this in-flight.

File: setup.py [hunk 3]
```diff
@@ -5,56 +5,76 @@
...
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3.4",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
+        "Topic :: Internet",
+        "Topic :: Internet :: WWW/HTTP",
+        "Topic :: Software Development :: Libraries",
+        "Topic :: Utilities",
+    ],
+)
```
Comment (jordiecometrica) [inline setup.py:82]: These cosmetic changes seem fairly unnecessary. Are they all cosmetic? Kind of hard to see in the hunk, but I assume they are. Cosmetic changes should be in a separate commit.

File: tox.ini [hunk 4]
```diff
@@ -7,8 +7,6 @@
 setenv=
```
Comment (jordiecometrica) [inline tox.ini]: Update how? Why is the section removed? Can you explain that in the commit message?

File: setup.py [hunk 5]
```diff
@@ -15,7 +15,7 @@
     "tox",
```
Comment (jordiecometrica) [inline setup.py:15]: Same comment as before. The commit message is uninformative.

File: setup.py [hunk 6]
```diff
@@ -15,7 +15,7 @@
...
 ]
 
-if sys.version_info < (3, 5):
+if sys.version_info < (3, 4):
```
Comment (jordiecometrica) [inline setup.py:18]: I'm confused by this change. Do the tests not work with Python 3.5 and those Django versions?

File: multisite/models.py [hunk 7]
```diff
@@ -20,7 +20,6 @@
```
Comment (jordiecometrica) [inline multisite/models.py:20]: The commit message is unrelated to the contents of this commit.

File: multisite/models.py [hunk 8]
```diff
@@ -20,7 +20,6 @@
 
 if django.VERSION < (2,):
     from django.utils.translation import ugettext_lazy as _
-    from django.utils.encoding import python_2_unicode_compatible
```
Comment (jordiecometrica) [inline multisite/models.py]: Was this meant to be squashed into an earlier commit?

File: tox.ini [hunk 9]
```diff
@@ -8,15 +8,11 @@ setenv=
     PYTHONPATH = {toxinidir}:{env:PYTHONPATH:}
```
Comment (jordiecometrica) [inline tox.ini:8]: Let's not have merge commits in a single PR if avoidable. Can we rebase so that this merge commit isn't necessary?

File: CHANGELOG.rst [hunk 10]
```diff
@@ -2,6 +2,11 @@
 Release Notes
 =============
 
+Unreleased
```
Comment (jordiecometrica) [inline CHANGELOG.rst:5]: I plan to release this soon after into PyPI so just bump the version number. I suppose 1.8.0 is a reasonable version.

File: setup.py [hunk 11]
```diff
@@ -5,56 +5,78 @@
...
+    author_email="leonid@shestera.ru",
+    maintainer="Ecometrica",
+    maintainer_email="dev@ecometrica.com",
+    url="http://github.com/ecometrica/django-multisite",
+    packages=find_packages(),
+    include_package_data=True,
+    package_data={"multisite": files},
+    install_requires=["tldextract>=1.2"],
+    setup_requires=["pytest-runner"],
+    tests_require=[
```
Comment (pauricthelodger) [inline setup.py:53]: I reckon you intended `tests_require=tests_require,` here? =)

**Comment by erikvw** [comment]
> I have some niggles with the contents and organisation of the commits in this PR. None of them are that important and the overall structure of the suggested changes seems correct to me. If you'd rather I make the changes, let me know and I can rewrite your commits like I did last time.

That would be be fine with me, thanks!

**Comment by jordiecometrica** [comment]
Thanks, I've rebased, slightly rewritten, and merged your commits.

https://github.com/ecometrica/django-multisite/commits/4cb28c7adfae2763d50b24f5c75c66c5484bbf47
